apps/workflows/jobs/refill-daily.ts

```diff
@@ -13,12 +13,16 @@ client.defineJob({
   }),
 
   run: async (_payload, io, _ctx) => {
+    const date = new Date(Date.now());
+    const today = date.getDate();
     const db = connectDatabase();
     const tb = new Tinybird(env().TINYBIRD_TOKEN);
-    const t = new Date();
-    t.setUTCHours(t.getUTCHours() - 24);
+    const tDay = new Date();
+    const tMonth = new Date();
+    tDay.setUTCHours(tDay.getUTCHours() - 24);
+    tMonth.setUTCMonth(tMonth.getUTCMonth() - 1);
 
-    const keys = await io.runTask("list keys", () =>
+    const dailyKeys = await io.runTask("list keys Daily", () =>
```

Comment:
we're no longer separating monthly and daily keys right? let's not introduce that naming then